cmd/jujud/agent/model/manifolds.go

```diff
@@ -216,6 +217,15 @@ func commonManifolds(config ManifoldsConfig) dependency.Manifolds {
 			Logger:        config.LoggingContext.GetLogger("juju.worker.credentialvalidator"),
 		}),
 
+		stateName := 
```

Comment:
I need to work out how to pass the state into the model worker. We've get this for free in 4.0